src/client/Microsoft.Identity.Client.Broker/Microsoft.Identity.Client.Broker.csproj

```diff
@@ -42,18 +45,8 @@
     <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNetDesktop)'">
```

Comment:
Why are these removed?

Reply:
For SDK projects, I think we should rely on the precompilation constants which are already defined by .NET, e.g. `if NET45_OR_HIGHER`. The whole `DESKTOP`, `WINDOWS_APP` naming is unfortunate. Can add back though, not a problem.